applications/StructuralMechanicsApplication/custom_constitutive/elastic_isotropic_3d.h

```diff
@@ -360,6 +360,15 @@ class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) ElasticIsotropic3D
         const ProcessInfo& rCurrentProcessInfo
         ) override;
 
+
+    virtual double GetMaterialProperty(
+        const Variable<double>& rVariable,
+        ConstitutiveLaw::Parameters& rValues
+        )
+    {
+        return rValues.GetMaterialProperties()[rVariable];
+    }
```

Comment:
If we allow this, then it could also be a semi-solution for #2349 (one would still need a derived and specialized CLaw but better than nothing)

So if the performance penalty is not too large I am positive about adding this!